### DISCUSSION TITLE
Started a deploy, cancelled the deploy, now I can't make a new deploy, it just returns "The deployment has been canceled."

### DISCUSSION THREAD

**Discussion by trevcodesjs** [vercel/vercel#4301] [General] [answered]
As title describes, I used `now` to start a deploy, canceled it on the Vercel dashboard because I wanted to deploy to prod instead. So then I tried running `now --prod` but it just returns `The deployment has been canceled.` in the CLI. It's been half an hour now and it still says that. How can I deploy again?

**Comment by cdock1029** [comment, marked as answer]
Did you try with `--force` ?
